docs/update_and_migration/from_4.3/update_from_4.3.md

```diff
@@ -0,0 +1,16 @@
+---
+description: Update your installation to the latest v4.4 version from v4.3.x.
+---
+
+# Update from v4.3.x to v4.4
+
+This update procedure applies if you are using the newest v4.3 installation.
+
+There are two routs you can take.
+The first one is for you, if you do not use old commerce packages, then you can proceed with removing them.
+The second one is for you if you use commerce packages and would like to continue to use them, even though they have deprecated since v4.4.
```

Comment:
Can we add explanation why there are two routes the begging ? Something like "ibexa/commerce-* packages are deprecated since 4.4.0 ..."

Reply:
We can add some additional explanation, the landing page with cards currently looks like that: 
![image](https://user-images.githubusercontent.com/35031941/215766397-69dd22c3-0ba0-47bc-ab83-b624a62cbdfa.png)